File: homework_1/12.draw_triangle/draw_triangle.cpp
#include <iostream>
using namespace std;

int main() {
    int side;
    cin >> side;

    for(int i = 1; i < side + 1; ++i){
        for(int j = 0; j < i; ++j){
            cout << "*";
        }
        cout << endl;
    }

    return 0;
}

File: homework_1/12.draw_triangle/draw_triangle.java
import java.util.Scanner;

class draw_triangle {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);

        int side = input.nextInt();

        for(int i = 1; i < side + 1; ++i){
            for(int j = 0; j < i; ++j){
                System.out.print("*");
            }
            System.out.println();
        }
   
        input.close();
    }
}